icons/electron.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Electron icon</title><path d="M3.1 11.8c.2-.1.2-.3.1-.4-.9-1.7-1.1-3.1-.5-4.2.8-1.4 3-2 5.9-1.4.2 0 .4-.1.4-.3 0-.2-.1-.3-.3-.4-3.1-.6-5.6 0-6.6 1.7-.7 1.3-.5 3 .5 4.8.1.2.3.3.5.2zM21.5 16.7c-.9 0-1.5.7-1.5 1.5 0 .3.1.5.2.8-2.2 1.2-6.6.5-10.8-1.9C7.6 16 6 14.8 4.8 13.4c-.1-.1-.3-.1-.5 0s-.1.3 0 .5c1.3 1.4 2.9 2.7 4.8 3.8 4.4 2.5 9.1 3.3 11.6 1.8.3.2.6.3.9.3.9 0 1.5-.7 1.5-1.5 0-.9-.7-1.6-1.6-1.6zm0 2.5c-.2 0-.4-.1-.6-.2l-.1-.1c-.1-.2-.2-.4-.2-.6 0-.5.4-.9.9-.9s.9.4.9.9c0 .4-.4.9-.9.9z"/><path d="M2.5 16.7h-.1c-.1-2.6 2.7-6.1 7-8.6 1.8-1.1 3.8-1.8 5.5-2.2.2 0 .3-.2.2-.4 0-.2-.2-.3-.4-.2-1.7.4-3.7 1.1-5.6 2.2-4.6 2.6-7.6 6.5-7.4 9.4-.5.3-.8.8-.8 1.3 0 .9.7 1.5 1.5 1.5S4 19.1 4 18.3c0-.9-.7-1.6-1.5-1.6zm0 2.5c-.5 0-.9-.4-.9-.9s.4-.9.9-.9.9.4.9.9c0 .4-.4.9-.9.9zM17.6 5.6c1.8 0 3.2.6 3.8 1.6.8 1.4.2 3.6-1.7 5.8-.1.1-.1.3 0 .5.1.1.3.1.5 0 2.1-2.4 2.8-4.9 1.8-6.6-.8-1.2-2.4-1.9-4.4-1.9-.2 0-.3.1-.3.3-.1.2.1.3.3.3zM15.7 20.8c-.2-.1-.3 0-.4.1-.9 1.6-2.1 2.4-3.3 2.4-1.6 0-3.2-1.6-4.2-4.4-.1-.2-.2-.3-.4-.2-.2.1-.3.2-.2.4C8.3 22.2 10 24 12 24c1.4 0 2.8-1 3.8-2.7.1-.2.1-.4-.1-.5z"/><path d="M12 3.1c.6 0 1.1-.3 1.4-.8 2.2 1.3 3.8 5.5 3.8 10.4 0 2.1-.3 4.1-.8 5.8-.1.2 0 .3.2.4.2.1.3 0 .4-.2.6-1.8.9-3.8.9-6 0-5.2-1.8-9.7-4.3-11v-.1C13.6.7 12.9 0 12 0c-.9 0-1.5.7-1.5 1.5 0 .9.7 1.6 1.5 1.6zm0-2.5c.5 0 .9.4.9.9s-.4.9-.9.9-.9-.4-.9-.9.4-.9.9-.9zM11.8 11.5c-.6.1-1 .7-.9 1.3.1.6.7 1 1.3.9.6-.1 1-.7.9-1.3-.1-.6-.7-1-1.3-.9z"/></svg>
```

Comment:
Two things about the SVG:

1. Looking at the quality of the icon it seems you've optimized a bit too much (look at the open circles at the bottom)
1. I think it might be better to use the version of the icon as it is seen in their favicon (i.e. with the open circles filled). What are your opinions @nexus0212 and @sh7dm?